services/motion/builtin/builtin_test.go

```diff
@@ -990,8 +1001,9 @@ func TestArmGantryPlanCheck(t *testing.T) {
 		worldState, err := referenceframe.NewWorldState(gifs, nil)
 		test.That(t, err, test.ShouldBeNil)
 
-		err = motionplan.CheckPlan(fs.Frame("xArm6"), plan, worldState, fs, startPose, inputs, errorState, logger)
+		collisionPose, err := motionplan.CheckPlan(fs.Frame("xArm6"), plan, worldState, fs, startPose, inputs, errorState, logger)
 		test.That(t, err, test.ShouldNotBeNil)
+		test.That(t, collisionPose, test.ShouldBeNil)
```

Comment:
Why is the collision pose nil? I thought we want to return the collision pose when a failure occurs.